index.js

```diff
@@ -0,0 +1,33 @@
+import BooksClass from './modules/Books.js';
+import dateTime from './modules/dateTime.js';
+
+dateTime();
+setInterval(dateTime, 1000);
+
+const addBtn = document.querySelector('#add');
+
+addBtn.addEventListener('click', () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  if (title === '' || author === '') {
+    // eslint-disable-next-line no-alert
+    alert('Please fill the required Fields');
+  } else {
+    const bookAd = new BooksClass();
+    bookAd.add();
+  }
+});
```

Comment:
- [ ] for [javascript best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) kindly avoid using the javascript `alert()` method, t’s ugly, impossible to style, it stops code execution and displays differently on different browsers. you can put a custom error div and display your messages or create a modal for it , please fix it 🙏